extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts

```diff
@@ -48,7 +50,7 @@ export class PublishDatabaseDialog {
 	public readPublishProfile: ((profileUri: vscode.Uri) => any) | undefined;
 
 	constructor(private project: Project) {
-		this.dialog = azdata.window.createModelViewDialog(constants.publishDialogName);
+		this.dialog = azdata.window.createModelViewDialog(constants.publishDialogName, 'publishDialog');
```

Comment:
For these names I'd probably be a little bit more specific - a name like `publishDialog` is too generic when looking at all model view dialogs.